### DISCUSSION TITLE
Timestamps from ElasticSearch

### DISCUSSION THREAD

**Discussion by benganellison** [marcusolsson/grafana-gantt-panel#44] [Q&A]
Just wanted to check if someone got timestamps from Elastic search to work properly.
I get it in UTC with the string "2021-06-30T07:55:22.344" in ElasticSearch it is defined as a timestamp. But somehow it is converted to local time. It should show 2021-06-30 09:55:22 in the UI but is showing 2021-06-30 07:55:22

**Comment by marcusolsson** [comment]
The timestamp you give as an example doesn't have any time zone information, the plugin doesn't know it's UTC. I suspect that's why the plugin assumes local time.

**Reply by benganellison** [reply]
strange thing is, when looking in kibana and json raw format it looks like this

```
  "ReferencemarkTimestamp": [
      "2021-07-05T08:57:49.589Z"
    ],
```
